src/librekey/rnp_key_store.cpp

```diff
@@ -438,6 +438,7 @@ rnp_key_store_add_subkey(rnp_key_store_t *keyring, pgp_key_t *srckey, pgp_key_t
         /* in case we already have key let's merge it in */
         if (!rnp_key_store_merge_subkey(oldkey, srckey, primary)) {
             RNP_LOG_KEY("failed to merge subkey %s", srckey);
+            RNP_LOG_KEY("primary key %s", primary);
```

Comment:
Let's do `"primary key is %s"` as in other places

Reply:
@rrrooommmaaa Thanks, fixed and force-pushed.